skymp5-server/cpp/server_guest_lib/WorldState.cpp

```diff
@@ -252,11 +253,11 @@ bool WorldState::AttachEspmRecord(const espm::CombineBrowser& br,
                                   const espm::IdMapping& mapping)
 {
   auto& cache = GetEspmCache();
-  auto refr = reinterpret_cast<espm::REFR*>(record);
+  auto* refr = espm::Convert<espm::REFR>(record);
```

Comment:
please keep using reinterpret_cast here and add a comment about why:
This is a hot path, we prefer reinterpret_cast since it doesn't generate any assembler code